[2024-12-22T14:56:32.703+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-22T14:56:32.760+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_top_best_sellers_books.create_date_dim_table scheduled__2024-12-15T00:00:00+00:00 [queued]>
[2024-12-22T14:56:32.833+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_top_best_sellers_books.create_date_dim_table scheduled__2024-12-15T00:00:00+00:00 [queued]>
[2024-12-22T14:56:32.843+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-22T14:56:32.938+0000] {taskinstance.py:2889} INFO - Executing <Task(PostgresOperator): create_date_dim_table> on 2024-12-15 00:00:00+00:00
[2024-12-22T14:56:32.980+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=143) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-22T14:56:33.009+0000] {standard_task_runner.py:72} INFO - Started process 147 to run task
[2024-12-22T14:56:33.011+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_top_best_sellers_books', 'create_date_dim_table', 'scheduled__2024-12-15T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/dag_test.py', '--cfg-path', '/tmp/tmpsi56b372']
[2024-12-22T14:56:33.045+0000] {standard_task_runner.py:105} INFO - Job 7: Subtask create_date_dim_table
[2024-12-22T14:56:34.103+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_top_best_sellers_books.create_date_dim_table scheduled__2024-12-15T00:00:00+00:00 [running]> on host c50092bbd7fa
[2024-12-22T14:56:34.963+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_top_best_sellers_books' AIRFLOW_CTX_TASK_ID='create_date_dim_table' AIRFLOW_CTX_EXECUTION_DATE='2024-12-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-15T00:00:00+00:00'
[2024-12-22T14:56:34.984+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-22T14:56:35.103+0000] {sql.py:278} INFO - Executing: CREATE TABLE IF NOT EXISTS date_dim (
    date_id SERIAL PRIMARY KEY,  -- Identifiant unique pour chaque date
    date DATE NOT NULL,          -- Date complète
    month INT NOT NULL,          -- Mois de l'année (1-12)
    year INT NOT NULL            -- Année (ex. 2024)
);
[2024-12-22T14:56:35.150+0000] {base.py:84} INFO - Retrieving connection 'books_connection'
[2024-12-22T14:56:35.187+0000] {base.py:84} INFO - Retrieving connection 'books_connection'
[2024-12-22T14:56:35.208+0000] {sql.py:553} INFO - Running statement: CREATE TABLE IF NOT EXISTS date_dim (
    date_id SERIAL PRIMARY KEY,  -- Identifiant unique pour chaque date
    date DATE NOT NULL,          -- Date complète
    month INT NOT NULL,          -- Mois de l'année (1-12)
    year INT NOT NULL            -- Année (ex. 2024)
);, parameters: None
[2024-12-22T14:56:35.290+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-22T14:56:35.296+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_top_best_sellers_books, task_id=create_date_dim_table, run_id=scheduled__2024-12-15T00:00:00+00:00, execution_date=20241215T000000, start_date=20241222T145632, end_date=20241222T145635
[2024-12-22T14:56:35.508+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-22T14:56:37.353+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-22T14:56:37.361+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
